frontend/src/components/custom-tools/header/Header.jsx

```diff
@@ -199,6 +227,21 @@ function Header({
           toolDetails={toolDetails}
         />
       </div>
+
+      {confirmModalVisible && (
+        <Modal
+          onOk={handleConfirmForceExport} // Pass the confirm action
+          onCancel={() => setConfirmModalVisible(false)} // Close the modal on cancel
+          open={confirmModalVisible}
+          title="Are you sure"
+          okText="Force Export"
+          centered
+        >
+          Unable to export tool. Some Prompt(s) were not run. Please run them
+          before exporting.{" "}
+          <strong>Would you like to force export anyway?</strong>
+        </Modal>
+      )}
```

Comment:
@jagadeeswaran-zipstack, why are we conditionally rendering the modal component? Every time the modal opens and closes, the component will get mounted and unmounted, creating unnecessary load. Wouldn't it be better to simply render it and let Antd's Modal component to take care of the visibility using `open` prop?

Reply:
This won't add any additional load to component but yes we can let antd handle this.